openapi/src/main/definitions/wacodis-schemas.yml

```diff
@@ -145,6 +145,25 @@ components:
               description: |
                 the id of the dataset within the GDI-DE catalogue
               type: string
+    DwdDataEnvelope:
+       allOf:
+        - $ref: '#/components/schemas/AbstractDataEnvelope'
+        - type: object
+          required:
```

Comment:
"serviceUrl" is required for idetifying a specific dataset and later generation of a WFS request.